src/Aspire.Hosting.Azure.ServiceBus/ApplicationModel/ServiceBusCorrelationFilter.cs

```diff
@@ -0,0 +1,67 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Aspire.Hosting.Azure.ServiceBus;
```

Comment:
(nit) can you match the folder name on disk and the namespace?

Reply:
Which one to keep, the disk or the namespace (already changed it once based on feedback, so I chose wrong apparently)

Reply:
Decide on the namespace. Then match it with the folders. That is my normal operation.